src/contribute/intellij.md

```diff
@@ -41,8 +41,8 @@ limitations under the License.
        * Create separate module per source set
        * Store generated project files externally
        * Use default gradle wrapper
-4. Delegate build actions to Gradle by going to Settings > Build, Execution,
```

Comment:
It seems `Settings/Preferences` is OS specific in linux there is no `Preferences` and since it is Settings in other lines maybe we let `Settings`. Apart of this LGTM.

Reply:
I believe this is also version dependent. In 2017.1 I believe it was `... > Build Tools > Gradle`

I can confirm that in 2018.1 it is now `... > Build Tools > Gradle > Runner`

How about we compromise on `Preferences/Settings > Build, Execution, Deployment > Build Tools > Gradle > Runner`?

Reply:
Yep. Also I'm testing this with 2017.3 - will mention the Intellij version here as well. Do we need to add anything regarding manual classpath tweaks?